ocaml/dream/default.nix

```diff
@@ -41,10 +41,10 @@ buildDunePackage {
   pname = "dream";
   version = "1.0.0-alpha2";
   src = fetchFromGitHub {
-    owner = "aantron";
+    owner = "EduardoRFS";
```

Comment:
why?

Reply:
This might not be needed anymore, I can test a bit later

Reply:
Reverted